collections/nemo_nlp/nemo_nlp/utils/callbacks/joint_intent_slot.py

```diff
@@ -92,12 +88,13 @@ def eval_epochs_done_callback(global_vars, graph_fold):
     i = 0
     if intent_preds.shape[0] > 21:
         i = random.randint(0, intent_preds.shape[0] - 21)
-    logger.info("Sampled i_preds: [%s]" % list2str(intent_preds[i:i+20]))
-    logger.info("Sampled intents: [%s]" % list2str(intent_labels[i:i+20]))
-    logger.info("Sampled s_preds: [%s]" % list2str(slot_preds[i:i+20]))
-    logger.info("Sampled slots: [%s]" % list2str(slot_labels[i:i+20]))
+    nemo.logging.info("Sampled i_preds: [%s]" % list2str(intent_preds[i:i+20]))
```

Comment:
debug?

Reply:
Same argument as above:
I would prefer logging examples during evaluation. It helps to see if your model is training.
Therefore, I would argue to keep it at info